### ISSUE TITLE
cannot import name '_prctl_pr_set_pdeathsig'

### ISSUE THREAD

**Issue by daiwk** [facebookresearch/pytext#115]
## Steps to reproduce

on my osx

```
pip install pytext-nlp
pip install caffe2
```

## Observed Results

when I run `pytext` it comes out:

```
Traceback (most recent call last):
  File "/home/work/anaconda3/bin/pytext", line 7, in <module>
    from pytext.main import main
  File "/home/work/anaconda3/lib/python3.6/site-packages/pytext/main.py", line 28, in <module>
    from torch.multiprocessing.spawn import spawn
  File "/home/work/anaconda3/lib/python3.6/site-packages/torch/multiprocessing/spawn.py", line 8, in <module>
    from . import _prctl_pr_set_pdeathsig
ImportError: cannot import name '_prctl_pr_set_pdeathsig'
```

**Comment by seayoung1112** [comment]
Could you try only 
`pip install pytext-nlp`

Caffe2 is merged with PyTorch 1.0, which is already in the dependency list. Manually installing Caffe2 might introduce a wrong version

**Comment by daiwk** [comment]
> Could you try only
> `pip install pytext-nlp`
> 
> Caffe2 is merged with PyTorch 1.0, which is already in the dependency list. Manually installing Caffe2 might introduce a wrong version

It works~
I tried to uninstall `caffe2` `pytext-nlp` and `torch`
then `pip install pytext-nlp`, then it works~
thank you very much~
